arviz/plots/plot_utils.py

```diff
@@ -385,6 +386,108 @@ def calculate_point_estimate(point_estimate, values, bw="default", circular=Fals
     return point_value
 
 
+def plot_point_interval(
+    ax, values, point_estimate, hdi_prob, quartiles, linewidth, markersize, rotated, interval_kwargs=None, backend='matplotlib'
+):
+
+    """ Plots point intervals
+    
+    Translates the data and represents them as point and interval summaries
+    
+    Parameters
+    ----------
+    ax : axes
+        Matplotlib axes
+    values : array-like
+        Values to plot
+    point_estimate : Optional[str]
+        Plot point estimate per variable. Values should be ‘mean’, ‘median’, ‘mode’ or None. 
+        Defaults to ‘auto’ i.e. it falls back to default set in rcParams.
+    linewidth : int
+        Line width throughout. If None it will be autoscaled based on figsize.
+    markersize : int
+        Markersize throughout. If None it will be autoscaled based on figsize.
+    hdi_prob : float
+        Valid only when point_interval is True. Plots HDI for chosen percentage of density. 
+        Defaults to 0.94.
+    rotated : bool
+        Whether to rotate the dot plot by 90 degrees.
+    interval_kwargs : dict
+        Keyword passed to the point interval
+    """
+
+    endpoint = 100 * (1 - hdi_prob) / 2
+    if quartiles:
+        qlist_interval = [endpoint, 25, 75, 100 - endpoint]
+    else:
+        qlist_interval = [endpoint, 100 - endpoint]
+    quantiles_interval = np.percentile(values, qlist_interval)
```

Comment:
Using `quantile` is a little easier to follow when the variables record quantiles:
```suggestion
    endpoint = (1 - hdi_prob) / 2
    if quartiles:
        qlist_interval = [endpoint, .25, .75, 1 - endpoint]
    else:
        qlist_interval = [endpoint, 1 - endpoint]
    quantiles_interval = np.quantile(values, qlist_interval)
```